.github/workflows/ci.yaml

```diff
@@ -100,8 +105,17 @@ jobs:
           node-version: 18.18.0
           cache: yarn
 
+      - name: Install rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: nightly-2023-04-17
+
       - name: Use era-test-node for testing
-        uses: dutterbutter/era-test-node-action@latest
+        uses: baptiste0928/cargo-install@v2
```

Comment:
where does this come from? can we trust it?

Reply:
So we're using external GitHub Actions anyway, in that sense I don't think we can trust `baptiste0928/cargo-install` any less than the others.

Reply:
But I raised this issue to @yorik, waiting for response from him!

Reply:
He suggested to use the commit SHA instead of the `@v2` tag, so I updated it!